cypress/support/pages/tasks/pageTasks.js

```diff
@@ -63,21 +63,27 @@ class Tasks {
   static #taskCardTitle = '[data-testid="taskTitle"]'
   static #taskMenuDelete = '[data-testid="task-delete"]'
   static #deleteTaskButton = '[data-testid="task-details-btn-delete"]'
+  static #videoConferenceNotStartedIcon = '[data-testid="video-conference-not-started-info-icon"]'
 
-  navigateToTasksOverview() {
+
+  seeVideoParticipationAsStudent () {
+    cy.get()
```

Comment:
I guess, here you forgot to add something

Reply:
this method is no longer required (moderator starts the conference in the test), so removed it.